src/main/kotlin/app/hashers/CommitObservable.kt

```diff
@@ -0,0 +1,114 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
+package app.hashers
+
+import app.Logger
+import app.model.Commit
+import app.model.DiffContent
+import app.model.DiffFile
+import app.model.DiffRange
+import app.model.Repo
+import app.utils.RepoHelper
+import io.reactivex.Observable
+import org.eclipse.jgit.api.Git
+import org.eclipse.jgit.diff.DiffEntry
+import org.eclipse.jgit.diff.DiffFormatter
+import org.eclipse.jgit.diff.RawText
+import org.eclipse.jgit.errors.MissingObjectException
+import org.eclipse.jgit.lib.ObjectId
+import org.eclipse.jgit.revwalk.RevCommit
+import org.eclipse.jgit.revwalk.RevWalk
+import org.eclipse.jgit.util.io.DisabledOutputStream
+import java.nio.charset.Charset
+
+object CommitObservable {
+    fun getObservable(git: Git, repo: Repo) = Observable
```

Comment:
It's a tautology to have CommitObservable.getObservable(). You may want to rename it to something like Commits.asObservable().

Reply:
Changed name to CommitCrawler, method name the same.